### PR TITLE
Add :LeanPlainTermGoal in case someone wants inline-hover term goals.

### PR DESCRIPTION
**Pull request by Julian**
Demo:

[asciicast](https://asciinema.org/a/7gZzqLn1JfoykRy23EAL5iIDw)

### CODE REVIEW

File: lua/lean/lsp.lua [hunk 1]
```diff
@@ -33,4 +40,18 @@ function M.handlers.plain_goal_handler (_, method, result, _, _, config)
...
+  config.focus_id = method
+  if not (result and result.goal) then
+    return
+  end
+  local markdown_lines = vim.lsp.util.convert_input_to_markdown_lines(result.goal)
+  markdown_lines = vim.lsp.util.trim_empty_lines(markdown_lines)
+  if vim.tbl_isempty(markdown_lines) then
+    return
+  end
+  return vim.lsp.util.open_floating_preview(markdown_lines, "leaninfo", config)
```
Comment (gebner) [inline lua/lean/lsp.lua:54]: Why is the goal converted to markdown if the filetype is leaninfo?

**Review by gebner — COMMENTED** [review]
This looks great!  There should probably be an entry in the README for the new command though.

File: lua/lean/lsp.lua (hunk 1 above)
Comment (Julian) [reply lua/lean/lsp.lua:54]: That's me bad-copypasting from the above. Will fix.

**Comment by Julian** [comment]
The README needs some love (which Patrick pointed out as well) -- I had some local changes on a start there, but for now, just added two lines on the two current commands so they're at least mentioned (and fixed the unnecessary markdownizing).

**Pull request merged**